src/wazuh_db/wdb.h

```diff
@@ -2168,7 +2169,7 @@ int wdb_global_reset_agents_connection(wdb_t *wdb, const char *sync_status);
  * @retval JSON with agents IDs on success.
  * @retval NULL on error.
  */
-cJSON* wdb_global_get_agents_by_connection_status (wdb_t *wdb, int last_agent_id, const char* connection_status, wdbc_result* status);
+cJSON* wdb_global_get_agents_by_connection_status (wdb_t *wdb, int last_agent_id, const char* connection_status, wdbc_result* status, char* node_name);
```

Comment:
Add new parameter to function description.

Reply:
Also, I think you can make `node_name` const.

Reply:
Done!